### ISSUE TITLE
Multiple selection on Assests\People\Licenese (basically anywhere you can select many objects)

### ISSUE THREAD

**Issue by avielc** [grokability/snipe-it#10606]
### Snipe-IT Version

5.3.7

### Operating System

CEntos

### Web Server

Nginx

### PHP Version

7.4

### Is your feature request related to a problem? Please describe. A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

When trying to make changes for multiple users, I can only select one at the time, 
Holding Shift and mouse click doesn't help with multiple selection (some UI on all kinds of apps offer to use shift for multi select so you can select 10-20 names at the time instead of click 10, 20 times or more. )

### Describe the solution you'd like A clear and concise description of what you want to happen.

I actually don't know what's the implementation required here, 
Basically just look at Gmail to understand the concept if you are unaware of it.,
- click to select an email in the checkbox next to the mail
- scroll down a couple of emails
- hold Shift and click the check box next to a mail that is 2 or more emails away from the first one select.
- results: all mails will be selected between the last and the first.

### Describe alternatives you've considered A clear and concise description of any alternative solutions or features you've considered.

If I'm working on most employees then just select all helps and then i de-select all the irrelevant users.
it's a bad alternative, but it's what I can work with.

**Comment by snipe** [comment]
Do the checkboxes not work for you?
<img width="532" alt="All_Assets____Snipe-IT_Demo" src="https://user-images.githubusercontent.com/197404/152253056-ed3bd28d-fc23-4471-83b1-59739247fe1f.png">

Not all things have a bulk edit option, but for those that do, using the checkboxes is how you select them

**Comment by avielc** [comment]
of course, but if I have a list of 500 employees where 250-350 of them are a mix of imported accounts from the CSV that just didn't clash well with LDAP Sync (duplicates) and some disabled accounts that got synced for the first time because I couldn't nail the LDAP filtering. 
Going through these many (which is probably not that bad compared to snipe users who hold multiple companies there)
is nerve-wracking, 
If I can click on one check box, scroll down a few  names that are all "not ldap synced"  and then Shift+click a checkbox to select all names - that would be perfect!
*update* (and also very intuitive for use)

**Comment by snipe** [comment]
It's really more of a limitation of the javascript table library we use. I'm not sure that's supported. I know there is a click-to-select extension, but not sure it can handle what you're asking :(
